src/ILLink.RoslynAnalyzer/Resources.resx

```diff
@@ -135,10 +135,10 @@
   <data name="AvoidAssemblyGetFilesInSingleFileMessage" xml:space="preserve">
     <value>'{0}' will throw for assemblies embedded in a single-file app</value>
   </data>
-  <data name="RequiresAssemblyFilesMessage" xml:space="preserve">
-    <value />
-  </data>
   <data name="RequiresAssemblyFilesTitle" xml:space="preserve">
-    <value />
+    <value>RequiresAssemblyFilesAnalyzer</value>
```

Comment:
I don't think this is supposed to be the title of the analyzer, it's the title of the warning, which is like a short description of what the warning is supposed to prevent.

Something like, "Avoid calling members marked with `RequiresAssemblyFilesAttribute` when publishing as a single file"?